api/http/handlerfuncs_export.go

```diff
@@ -0,0 +1,188 @@
+// Copyright 2023 Democratized Data Foundation
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.txt.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0, included in the file
+// licenses/APL.txt.
+
+package http
+
+import (
+	"net/http"
+
+	"github.com/fxamacker/cbor/v2"
+
+	"github.com/sourcenetwork/defradb/client"
+)
+
+func exportHandler(rw http.ResponseWriter, req *http.Request) {
```

Comment:
suggestion: It does feel a bit odd that this is only exposed over the http client, and not via the `client` package too.  It might be nicer if the bulk of the code was pulled out into its own package too (regardless), for greater visibility and to make it easier to expose via other locations at a later date should we want to.

Reply:
I'm ok with moving it to the client package but I don't think its worth having it's own package.